nominees/akita.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Akita",
+  "aliases": [
+    ""
+  ],
+  "description": "A browser extension that gives you insight into your engagement with Web Monetization. Web Monetization is an emerging idea that’s paving the way for a more open, fair and inclusive web, to better support both users and creators. Web Monetization provides a way for micropayments to be streamed anonymously from users to websites.  In May 2020, we learned about Web Monetization. A few keywords stood out to us: open, fair and inclusive. These words weren’t something we were used to seeing amongst monetization on the web, which generally involves targeted ads, paywalls and other unfortunate things.  But we found ourselves struggling to understand and find resources to learn about Web Monetization. To tackle this problem, we began the Akita project. We are creating resources which introduce Web Monetization through storytelling, easy-to-use tools and community outreach.",
+  "website": "https://akitaproject.site",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/esse-dev/akita/blob/master/LICENSE"
```

Comment:
@nathanbaleeta, @lacabra - I see there are indeed 2 licenses in this one file. I am not sure if that is valid or not? Should the MIT and CC license be separated?

Reply:
I think it's fine, and would label the separation of licenses as out of scope. While I certainly prefer to have different licenses in different files (and that's how I have them in the repos I'm involved), I don't think there is necessarily a hard standard on this. I'm ok with it as long as the file contains the right text for each license, which is the case.